### DISCUSSION TITLE
Break

### DISCUSSION THREAD

**Discussion by bf-bonkus** [bf-bonkus/breadiquette#13] [Announcements]
What's up gamers.

Because the end of the term is coming up, I need to make sure to get my grades as high as possible (you guys might too, but you guys also generally have more time than me). This means I might not commit anything this week. @RuinKnight , I think at this point it might be better for you to just start messing around in your fork of the repo (make a new branch first.) @smithjac090 , I want you to make variations of the bread_guy sprite I have so that it'll be easy to fill out the French Bread Institute once I start the area.

See you guys next week!

**Comment by RuinKnight** [comment]
See you next week
